### ISSUE TITLE
KeyboardAwareScrollView not working properly with TextInput when used with states

### ISSUE THREAD

**Issue by sujay-bidchat** [APSL/react-native-keyboard-aware-scroll-view#214]
Try using **TextInput** with its value stored in **state** i.e for example -
<KeyboardAwareScrollView>
                              <TextInput
                                    onChangeText={(text) => {
                                        this.setState({
                                            myText: text
                                        });
                                    }}
                                    value={this.state.myText}
                                />
</<KeyboardAwareScrollView>

_After typing some characters try to scroll, the TextInput goes behind keyboard and user cannot scroll it back to top making it impossible for the user to see what is being typed._

**Comment by mateoc10** [comment]
@sujay-bidchat  I am having a very similar problem, did you find a solution?

**Comment by sujay-bidchat** [comment]
@mateoc10 Workaround for this is to close/hide the keyboard once the user tries to scroll/click outside.

**Comment by sujay-bidchat** [comment]
@alvaromb Btw I'm using version: 0.4.1

**Comment by 4external** [comment]
Hi!

I using react-native-keyboard-aware-scroll-view@0.4.3 and cannot reproduce.
Tested on Android 5 and ios 10.
Me text Input

```
			<TextInput
				underlineColorAndroid = {'transparent'}	
				style={styles.textInput}
				  
				onChangeText={(text) =>{ this.setState({whom:text})  }}
				placeholder='Whom'
				placeholderTextColor = '#7cbeda'
				autoGrow={true}
				multiline = {true}					
				value={this.state.whom}
			/>
```

**Comment by terryttsai** [comment]
I had this issue too on 0.4.1, but it was resolved when I updated to 0.4.4.
